ui/tract-renderer/src/main/res/layout/tract_content_input.xml

```diff
@@ -9,5 +9,6 @@
         android:id="@+id/input"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:background="@android:color/transparent"
```

Comment:
I want to take a look at the material components theming, this may not be the correct way to do this with the material components